app/src/main/res/layout/dialog_add_to_playlist.xml

```diff
@@ -14,5 +14,25 @@
         android:layout_marginTop="8dp"
         app:icon="@drawable/ic_playlist_add"
         tools:ignore="RtlSymmetry" />
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="24dp"
+        android:padding="8dp"
+        android:gravity="right|center_vertical"
```

Comment:
```suggestion
        android:gravity="end|center_vertical"
```
In order to also support `right to left (rtl)` screens, we have to use `end` instead of `right`.